streams/src/main/java/org/apache/kafka/streams/kstream/Printed.java

```diff
@@ -31,9 +31,10 @@
  * @param <V> value type
  * @see KStream#print(Printed)
  */
-public class Printed<K, V> {
+public class Printed<K, V> implements NamedOperation<Printed<K, V>> {
```

Comment:
@bbejeck If we add a method `Printed.as` then currenlty there is not method `withFile` or `withSysOut` to set the the outputstream.

I think we should not add the `Printed.as` method and update the KIP ?

Reply:
That sounds good to me, and if I recall correctly we had similar reasons for not adding `as` to `Suppressed`.

\cc @guozhangwang @mjsax @vvcephei @ableegoldman

Reply:
SGTM.

If we update the KIP, we should send a follow up email to the VOTE thread summarizing the changes (we can do this after all PRs are merged in case there is more)